src/app/shared/comcol-forms/comcol-form/comcol-form.component.html

```diff
@@ -12,11 +12,13 @@
           <div class="col-4 d-inline-block">
             <div *ngIf="logo" class="btn-group btn-group-sm float-right" role="group">
               <button *ngIf="!markLogoForDeletion" type="button" class="btn btn-danger"
+                      [attr.aria-label]="(type.value + '.edit.logo.delete.title' | translate)"
```

Comment:
This line & line 21 in this same file both are not working as expected.  The `aria-label` never appears on the `<button>` when you inspect the final page.  I also notice that you already had a `title` attribute on the `<button>`, which I think would **fix this problem** without the need for an `aria-label`.  But, that `title` attribute also never appears in the final code.  Maybe there's a bug in the `translate` line?

Reply:
@tdonohue 
I had a look and if I inspect the html code tith the devtools I can see both the title and the aria.label property
![Schermata da 2021-07-19 16-23-14](https://user-images.githubusercontent.com/2486489/126176290-e07787e4-ab19-4a45-a5f6-0722371631d9.png)

Moreover if I scan the page with the "axe DevTools" I don't get the accebility error, while If I remove bothe title and aria.label, I get the accessibility error "Buttons must have discernible text".

Reply:
@atarix83 : I checked again, and it looks like we are testing slightly different scenarios. You are correct, the `title` and `aria.label` property **do appear after clicking Save**.  But, they are **missing after the initial upload**.  Here's what I see, step by step..

1. Login as an Admin
2. Go to a Collection _which doesn't have a logo_
3. Edit the Collection. Drag & drop a logo.  Trash can icon appears (DO NOT CLICK SAVE YET)
4. Inspect the page... you'll see the Trash can `<button>` has no `title` nor `aria-label`
5. Now click Save. You go back to the Collection page
6. Click Edit.  Inspect the page.  Now, the Trash can `<button>` has both the `title` and the `aria-label`.

So, it seems I was testing immediately after upload, but you are correct the label & title do appear after saving.   Ideally, the attributes should appear in **both scenarios**.  But, if we cannot figure out a way to easily do so, this PR is likely "good enough" for now.

Reply:
@tdonohue yeah we're testing different scenarios. Now it should be fixed. I've removed unnecessary `aria-label`

Reply:
That fixed it! Thanks @atarix83 !  I'll go ahead and merge this now.